libraries/botbuilder-schema/tests/test_activity.py

```diff
@@ -0,0 +1,667 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import aiounittest
+from botbuilder.schema import (
+    Activity,
+    ConversationReference,
+    ConversationAccount,
+    ChannelAccount,
+    Entity,
+    ResourceResponse,
+    Attachment,
+)
+from botbuilder.schema._connector_client_enums import ActivityTypes
+
+
+class TestActivity(aiounittest.AsyncTestCase):
+    def test_constructor(self):
+        # Arrange
+        activity = Activity()
+
+        # Assert
+        self.assertIsNotNone(activity)
+        self.assertIsNone(activity.type)
+        self.assertIsNone(activity.id)
+        self.assertIsNone(activity.timestamp)
+        self.assertIsNone(activity.local_timestamp)
+        self.assertIsNone(activity.local_timezone)
+        self.assertIsNone(activity.service_url)
+        self.assertIsNone(activity.channel_id)
+        self.assertIsNone(activity.from_property)
+        self.assertIsNone(activity.conversation)
+        self.assertIsNone(activity.recipient)
+        self.assertIsNone(activity.text_format)
+        self.assertIsNone(activity.attachment_layout)
+        self.assertIsNone(activity.members_added)
+        self.assertIsNone(activity.members_removed)
+        self.assertIsNone(activity.reactions_added)
+        self.assertIsNone(activity.reactions_removed)
+        self.assertIsNone(activity.topic_name)
+        self.assertIsNone(activity.history_disclosed)
+        self.assertIsNone(activity.locale)
+        self.assertIsNone(activity.text)
+        self.assertIsNone(activity.speak)
+        self.assertIsNone(activity.input_hint)
+        self.assertIsNone(activity.summary)
+        self.assertIsNone(activity.suggested_actions)
+        self.assertIsNone(activity.attachments)
+        self.assertIsNone(activity.entities)
+        self.assertIsNone(activity.channel_data)
+        self.assertIsNone(activity.action)
+        self.assertIsNone(activity.reply_to_id)
+        self.assertIsNone(activity.label)
+        self.assertIsNone(activity.value_type)
+        self.assertIsNone(activity.value)
+        self.assertIsNone(activity.name)
+        self.assertIsNone(activity.relates_to)
+        self.assertIsNone(activity.code)
+        self.assertIsNone(activity.expiration)
+        self.assertIsNone(activity.importance)
+        self.assertIsNone(activity.delivery_mode)
+        self.assertIsNone(activity.listen_for)
+        self.assertIsNone(activity.text_highlights)
+        self.assertIsNone(activity.semantic_action)
+        self.assertIsNone(activity.caller_id)
+
+    def test_apply_conversation_reference(self):
```

Comment:
Here is missing the test when _isComming_ = true.

Reply:
added, thanks!